### PR TITLE
Add timestamp in autofill app state

### LINKED ISSUE #3438: Autofill data will not update display after editing instantly
**Did you search for similar issues before submitting this one?**
Yes
**Describe the issue you encountered:**
autofill data will not update display after editing instantly

- Brave Version: 0.11.6

- Steps to reproduce:
    1. Add a credit card 
    2. Edit credit card's expiration date
    3. Must refresh or switch other tab and back to see the update

- Any related issues: #3358

### PR DESCRIPTION
**Pull request by darkdh**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

fix #3438

### CODE REVIEW

File: js/components/frame.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ class Frame extends ImmutableComponent {
     } else if (location === 'about:autofill') {
       const partition = FrameStateUtil.getPartition(this.frame)
       if (this.props.autofillAddresses) {
-        const addresses = this.props.autofillAddresses.toJS()
+        const addresses = this.props.autofillAddresses.toJS().guid
```
Comment (bridiver) [inline js/components/frame.js]: can we keep this immutable? `autofillAddresses.get('guid')`

Comment (darkdh) [inline js/components/frame.js]: Fixed in fe8e3da

**Comment by darkdh** [comment]
sorry my bad

**Comment by bridiver** [comment]
++

**Pull request merged**